experimental/traffic-portal/src/styles.scss

```diff
@@ -100,7 +102,122 @@ body {
 	overflow-x: hidden;
 }
 
-.mat-mdc-button-base {
+:root {
+	--toolbar-height: 3em;
+	--sidebar-max-width: 25vw;
+	--content-max-width: calc(100vw - 5vw - var(--sidebar-max-width));
+}
+
+.small-icon-button {
+	width: 24px !important;
+	height: 24px !important;
+	padding: 0px !important;
+	align-items: center;
+	justify-content: center;
+	display: inline-flex !important;
+
+	& > *[role=img] {
+		width: 16px !important;
+		height: 16px !important;
+		font-size: 16px !important;
+
+		svg {
+			width: 16px !important;
+			height: 16px !important;
+		}
+	}
+
+	.mat-mdc-button-touch-target {
+		width: 24px !important;
+		height: 24px !important;
+	}
+}
```

Comment:
gosh that's an awful lot of `!important` - is that really necessary? And if so: why?

Reply:
Because of how the material CSS is encaspulated. If it's put in the component scss it would work but this is going to be needed else where. I added a mixin that should alleviate the issue